crates/bvs-vault-cw20-tokenized/src/contract.rs

```diff
@@ -0,0 +1,708 @@
+use cosmwasm_std::to_json_binary;
+use cosmwasm_std::{entry_point, Deps, DepsMut, Env, MessageInfo, Response, StdResult};
+use cw2::set_contract_version;
+use cw20_base::contract::instantiate as base_instantiate;
+use cw20_base::msg::InstantiateMsg as ReceiptCw20InstantiateMsg;
+
+use crate::error::ContractError;
+use crate::msg::{ExecuteMsg as CombinedExecuteMsg, InstantiateMsg, QueryMsg};
+use bvs_vault_cw20::token as PrimaryStakingToken;
```

Comment:
```suggestion
use bvs_vault_cw20::token as UnderlyingToken;
```

The convention here should be underlying.
"PrimaryStaking" makes it more confusing